src/components/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+import { action } from '@storybook/addon-actions';
+
+import Modal from './Modal';
+import { ModalProps } from './Modal.types';
+import { ModalWidthSizes } from './Modal.enums';
+import { H2 } from '../typography';
+import { Inline, Padbox } from '../layout';
+import { Button, ButtonEnums } from '../Button';
+import { generateControl } from '../../utils/tests/storybook';
+import { SpaceSizes } from '../../index';
+
+export default {
+  title: 'components/modals/Modal',
+  component: Modal,
+  argTypes: {
+    width: {
+      ...generateControl('select', ModalWidthSizes),
+    },
+    footer: {
+      control: { disable: true },
+    },
+  },
+} as Meta;
+
+const Footer = () => (
+  <Inline gap={SpaceSizes.md} justify="end">
```

Comment:
```suggestion
  <Inline gap={SpaceSizes.md} justify="flex-end">
```

This will fix footer button alignments in visual tests
cc @abykovssc